### PR TITLE
add libvirt dependency to Makefile

### PR DESCRIPTION
**Pull request by paulfantom**
Solution to run everything in containers (needed for CI).

Examples are implemented in https://github.com/openshift/release/pull/1415

/CC: @enxebre @petr-muller

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,14 +48,15 @@ push:
 
 check: fmt vet
 
-test:
+test: libvirt
```
Comment (enxebre) [inline Makefile]: You are setting CONTAINERIZED=1 here https://github.com/openshift/release/pull/1415/files#diff-8494a5bc314f945b17df41d8bb54cfc9R32
So CGO_deps won't execute so woulnd't this still fail?

File: Makefile [hunk 2]
```diff
@@ -16,13 +16,18 @@
...
+CONTAINERIZED ?= 0
+
 depend:
 	dep version || go get -u github.com/golang/dep/cmd/dep
 	dep ensure
 
 depend-update:
 	dep ensure -update
 
+libvirt:
```
[the remark this answers is not in the document]
Comment (paulfantom) [reply Makefile]: why so convoluted naming scheme?

File: Makefile (hunk 1 above)
Comment (paulfantom) [reply Makefile]: You are right. I fixed condition so it should work now.

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulfantom) [reply Makefile]: I changed it to sth more in line with the rest of Makefile

File: Makefile [hunk 3]
```diff
@@ -16,13 +16,18 @@
...
+CONTAINERIZED ?= 0
+
 depend:
 	dep version || go get -u github.com/golang/dep/cmd/dep
 	dep ensure
 
 depend-update:
 	dep ensure -update
 
+deps-cgo:
```
Comment (enxebre) [inline Makefile]: why CONTAINERIZED?

**Pull request merged**